src/services/ogc-features/src/main/resources/application.yml

```diff
@@ -63,3 +63,8 @@ postgis:
     #exclude:
     #- public
     #- tiger_data
+---
+spring:
+  config:
+    # See https://github.com/georchestra/datadir/tree/master/gateway for a configuration override example
+    import: optional:file:${georchestra.datadir}/data-api/application.yaml
```

Comment:
might require to set the GEORCHESTRA_DATADIR environment variable, or pass `-Dgeorchestra.datadir=/etc/georchestra` to the JVM. I don't know how it is currently done by the gateway